source/installation/install-docker.txt

```diff
@@ -0,0 +1,113 @@
+
+About this Task
+---------------
+
+Docker volumes can be used for persisting data when necessary in the event 
+that a container stops running or is restarted. For more information, see 
+`Volumes <https://docs.docker.com/storage/volumes/>`__.
+
+Before you Begin
+----------------
+
+You must have Docker installed on your computer. For more information, see  
+`Install Docker Engine <https://docs.docker.com/engine/install/>`__. 
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install a Kafka cluster
+
+      Use the following ``docker-compose`` example to install a Kafka cluster.
+      
+      You can also use an existing cluster and provide configuration details to connect 
+      your existing cluster.
+
+      .. code-block::
+
+         version: '3'
+         services:
+           kafka-1:
+           image: quay.io/debezium/kafka:2.2
+           ports:
+             - 19092:9092
+             - 19093:9093
+           environment:
+             CLUSTER_ID: oh-sxaDRTcyAr6pFRbXyzA
+             BROKER_ID: 1
+             KAFKA_CONTROLLER_QUORUM_VOTERS: 1@kafka-1:9093
+             NODE_ROLE: controller,broker
+           healthcheck:
+             test: ["CMD-SHELL", "./bin/kafka-topics.sh --version"]
+             interval: 5s
+             timeout: 10s
+             retries: 10
+
+   .. step:: Install a Kafka Connect cluster
+
+      Install a Kafka Connect cluster with the ``relational-migrator-connect`` image 
+      from `public Amazon ECR <public.ecr.aws/v4d7k6c9/relational-migrator:latest>`__.
+
+   .. step:: Install Relational Migrator
+
+      You have two options for installing Relational Migrator:
+
+      1. From the `download center <https://www.mongodb.com/try/download/relational-migrator>`__, 
+         select the appropriate Relational Migrator installer.
+
+      #. Create a docker image using a Relational Migrator installer.
+
+         For a complete example, you can download the following docker-compose: 
+         `Containerised kafka-connect cluster with Relational Migrator 
+         <https://migrator-installer-repository.s3-ap-southeast-2.amazonaws.com/1.4.3/docker-compose-migrator-single-kafka.yml>`__.
+
+   .. step:: If your source database is MySQL or Oracle, add their related drivers
+
+      Relational Migrator uses the JDBC driver of the respective source database to read 
+      database schema. It bundles SQL Server and PostgreSQL JDBC drivers. For MySQL and 
+      Oracle, you must add their drivers to the following path:
+
+      ``<path-to-installation>/mongodb-relational-migrator/lib/app/lib``
+      
+      - Oracle: `21.6.0.0 of odjbc11.jar from the Oracle 21c
+        <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`__
+      - MySQL: `latest 8.0.x version, Platform Independent Version 
+        <https://dev.mysql.com/downloads/connector/j/>`__
+
+
+Learn More
+----------
+
+- :ref:`kafka-intro`
```

Comment:
[For reviewer]

These currently cause build errors but they will be resolved once https://github.com/mongodb/docs-relational-migrator/pull/96 is merged.